graphics/macgui/macwindow.cpp

```diff
@@ -144,14 +150,15 @@ bool MacWindow::draw(Graphics::ManagedSurface *g, bool forceRedraw) {
 	_contentIsDirty = false;
 
 	// Compose
-	_composeSurface.blitFrom(_surface, Common::Rect(0, 0, _surface.w - 2, _surface.h - 2), Common::Point(2, 2));
```

Comment:
I don't think so, although I only tested dragging, resizing and writing on windows, not by actually playing the game. Although I can revert if necessary.